integrations/langfuse/README.md

```
@@ -0,0 +1,163 @@
+memanto migrate langfuse --discover     # scores, latency spread, error labels
+```
+
+```python
+@observe()
+def generate(prompt):
+    raise ValueError("Model returned malformed output")
+```
+
+produces a memory:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)